health/health.d/dockerd.conf

```diff
@@ -4,5 +4,5 @@ template: docker_unhealthy_containers
    every: 10s
   lookup: average -10s
     crit: $this > 0
-    info: number of unhealthy containers
+    info: average number of unhealthy docker containers over the last 10 seconds
```

Comment:
Should we add an explanation for the `unhealthy` here? :thinking: Because we are sending message because the system is with some problem, so it is implicit that the system is not health.

Reply:
It is https://docs.docker.com/engine/reference/builder/#healthcheck

> When a container has a healthcheck specified, it has a health status in addition to its normal status. This status is initially starting. Whenever a health check passes, it becomes healthy (whatever state it was previously in). After a certain number of consecutive failures, it becomes unhealthy.

Reply:
i mean if you want to suggest something, please do and i submit changes 😉